### ISSUE TITLE
test failed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jleinonen/pymiecoated#5]
```
What steps will reproduce the problem?
1. Clone with git
2. python setup.py install
3. Run the test.mie as instructed in the Instruction

The test failed with the following information:
FAIL: test_magnetic (pymiecoated.test.test_mie.MieTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "pymiecoated/test/test_mie.py", line 108, in test_magnetic
    self.assertLess(abs(S12_ref[0]-S12[0])/S12_ref[0], epsilon)
AssertionError: (4.4758213571384395e-14+5.3281104443037928e-15j) not less than 
2.220446049250313e-14

What version of the product are you using? On what operating system?

I'm using Ubuntu 14.04 system with anaconda distribution of python. 
```

Original issue reported on code.google.com by `hfyang...@gmail.com` on 24 Apr 2015 at 4:37

**Comment by orkan-tief** [comment]
The epsilon which defines if the test is passed is definend in test_mie.py Line 30:
`epsilon = 1e2*sys.float_info.epsilon`
The multiplication with 100 seem a bit arbitrary for me. If this value is increased slightly, the test is passed.

**Comment by jleinonen** [comment]
Fixed in 7ca9950.
